politeiawww/proposals.go

```diff
@@ -838,6 +890,59 @@ func (p *politeiawww) processProposalDetails(propDetails www.ProposalsDetails, u
 	return &reply, nil
 }
 
+// processBatchProposals fetches a list of proposals from the records
+// cache and returns them
```

Comment:
```suggestion
// cache and returns them. The returned proposals do not include the
// proposal files.
```

This is non-obvious behavior so its better to explicitly state it.